Library/Homebrew/utils/repology.rb

```diff
@@ -76,7 +76,8 @@ def parse_api_response(limit = nil, last_package = "", repository:)
       break if (limit && outdated_packages.size >= limit) || response.size <= 1
     end
 
-    puts "#{outdated_packages.size} outdated #{package_term.pluralize(outdated_packages.size)} found"
+    package_term = package_term.chop if outdated_packages.size == 1
```

Comment:
This was the only dynamic instance of `pluralize`. I took advantage of the fact that there's an easier path from plural to singular among {formulae, casks, packages} and reversed the logic here